findspam.py

```diff
@@ -397,7 +397,7 @@ def similar_answer(post):
             sanitized_answer = strip_urls_and_tags(other_answer.body)
             ratio = similar_ratio(sanitized_body, sanitized_answer)
 
-            if ratio >= SIMILAR_THRESHOLD:
+            if ratio >= 0.7:
```

Comment:
@ArtOfCode- @teward Because `similar_username` uses that constant and it would be way too low for it.